### PR TITLE
Add check for open level

### LINKED ISSUE #265: URDF Importer, prevent user from importing URDF without opened level
The current URDF importer crashes when the User tries to import URDF without the level opened.
Please find a way to prevent users from crashing in such a situation.

### PR DESCRIPTION
**Pull request by arturkamieniecki**
I've added a check for an open level while importing a URDF file. If a level is not open this check will prevent further pages in Robot Importer Widget, displaying a message box describing the error. This fixes #265.

### CODE REVIEW

**Comment by pawelbudziszewski** [comment]
It works and looks fine, but change the target to stabilization.

File: Gems/ROS2/Code/Source/RobotImporter/RobotImporterWidget.cpp [hunk 1]
```diff
@@ -94,13 +94,14 @@ namespace ROS2
                 else
                 {
                     report += "# " + tr("XACRO parsing failed") + "\n";
-                    report += "\n\n## " + tr("Command called") +  "\n\n`" + QString::fromUtf8(outcome.m_called.data()) + "`";
+                    report += "\n\n## " + tr("Command called") + "\n\n`" + QString::fromUtf8(outcome.m_called.data()) + "`";
```
Comment (michalpelka) [inline Gems/ROS2/Code/Source/RobotImporter/RobotImporterWidget.cpp:97]: Remove unnecessary changes.

File: Gems/ROS2/Code/Source/RobotImporter/RobotImporterWidget.cpp [hunk 2]
```diff
@@ -94,13 +94,14 @@ namespace ROS2
...
                     report += "\n\n" + tr("Process failed");
                     report += "\n\n## " + tr("Error output") + "\n\n";
                     report += "```\n";
                     if (outcome.m_logErrorOutput.size())
                     {
-                        report += QString::fromLocal8Bit(outcome.m_logErrorOutput.data(), static_cast<int>(outcome.m_logErrorOutput.size()));
+                        report +=
```
Comment (michalpelka) [inline Gems/ROS2/Code/Source/RobotImporter/RobotImporterWidget.cpp:103]: It was formatting? If not please remove it.

**Review by michalpelka — COMMENTED** [review]
Looks, good, please adjust.

**Comment by adamdbrw** [comment]
This change would be useful, thank you! Best to run clang-format, or integrate formatting into your IDE, to make sure style is followed.
cmd line: `find . -iname *.h -o -iname *.cpp | xargs clang-format-13 -i`

File: Gems/ROS2/Code/Source/RobotImporter/RobotImporterWidget.cpp (hunk 1 above)
Comment (arturkamieniecki) [reply Gems/ROS2/Code/Source/RobotImporter/RobotImporterWidget.cpp:97]: This was caused by my IDEs auto formatting. I'll revert it.

File: Gems/ROS2/Code/Source/RobotImporter/RobotImporterWidget.cpp (hunk 2 above)
Comment (arturkamieniecki) [reply Gems/ROS2/Code/Source/RobotImporter/RobotImporterWidget.cpp:103]: This was caused by my IDEs auto formatting. I'll revert it.

File: Gems/ROS2/Code/Source/RobotImporter/RobotImporterWidget.cpp [hunk 3]
```diff
@@ -245,7 +247,7 @@ namespace ROS2
             {
                 m_params = Utils::xacro::GetParameterFromXacroFile(m_urdfPath.String());
                 AZ_Printf("RobotImporterWidget", "Xacro has %d arguments\n", m_params.size())
-                m_xacroParamsPage->SetXacroParameters(m_params);
+                    m_xacroParamsPage->SetXacroParameters(m_params);
```
[the remark this answers is not in the document]
Comment (arturkamieniecki) [reply Gems/ROS2/Code/Source/RobotImporter/RobotImporterWidget.cpp:250]: This was caused by my IDEs auto formatting. I'll revert it.

**Comment by adamdbrw** [comment]
Please rebase on stabilization/2305 branch - there should be no extra commits in the PR
